### DISCUSSION TITLE
Query builder - order by desc example

### DISCUSSION THREAD

**Discussion by ediskandarov-cur** [edgedb/edgedb-js#501] [Q&A] [answered]
Hi there,

Could not find any examples on how to order by column in descendant order, with using query builder.

**Comment by Sikarii** [comment, marked as answer]
This can be done using the `order_by` property in *shapes*.
<https://www.edgedb.com/docs/clients/js/select#ordering>

**Reply by ediskandarov-cur** [reply]
Thanks, I was looking on another documentation page.
